vault/metadata.csv

```diff
@@ -248,8 +248,8 @@ vault.vault.merkle.savecheckpoint.sum,gauge,,millisecond,,Total time taken to sa
 vault.vault.wal.deletewals.count,count,,,,Number of samples of vault.vault.wal.deletewals.,0,vault,,
 vault.vault.wal.deletewals.quantile,gauge,,millisecond,,Time taken to delete a Write Ahead Log (WAL) quantile.,0,vault,,
 vault.vault.wal.deletewals.sum,gauge,,millisecond,,Total time taken to delete a Write Ahead Log (WAL).,0,vault,,
-vault.vault.wal.gc.deleted,count,,unit,,Number of Write Ahead Logs (WAL) deleted during each garbage collection run.,0,vault,,
-vault.vault.wal.gc.total,count,,unit,,Total Number of Write Ahead Logs (WAL) on disk.,0,vault,,
+vault.vault.wal.gc.deleted,gauge,,unit,,Number of Write Ahead Logs (WAL) deleted during each garbage collection run.,0,vault,,
+vault.vault.wal.gc.total,gauge,,unit,,Total Number of Write Ahead Logs (WAL) on disk.,0,vault,,
```

Comment:
These metrics are collected as gauges rather than counts: https://www.vaultproject.io/docs/internals/telemetry#merkle-tree-and-write-ahead-log-metrics

Reply:
What do you think about renaming it to something different to avoid confusion since it's never been collected before ie https://github.com/DataDog/integrations-core/blob/master/vault/tests/test_integration.py#L89

Reply:
Ah that would be a good idea to consider, although this was already given to a customer as a beta wheel so I think renaming now would be more confusing.